omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_checks.rb

```diff
@@ -28,6 +28,17 @@ def fail_with error
     end
   end
 
+  def first_run?
```

Comment:
I'm :+1: with the suggested change.  I think we made it super-explicit to keep things clear as we talked through it, and didn't revisit it before finishing.